crates/tx-parser/src/inscription.rs

```diff
@@ -19,8 +21,8 @@ pub enum InscriptionParseError {
     #[error("Invalid/Missing envelope(NO OP_IF..OP_ENDIF): ")]
     InvalidEnvelope,
     /// Does not have a valid name tag
-    #[error("Invalid/Missing name tag")]
-    InvalidNameTag,
+    #[error("Invalid/Missing Batch tag")]
+    InvalidBatchTag,
```

Comment:
This and `InvalidBlobTag` don't make sense anymore.  We can treat them both as `UnknownTag` since we have multiple tags we're paying attention to now in the simplified format.